templates/repo/issue/view_content/comments.tmpl

```diff
@@ -366,7 +366,7 @@
 					{{if not .OriginalAuthor}}
 					{{/* Some timeline avatars need a offset to correctly align with their speech bubble.
 						The condition depends on whether the comment has contents/attachments or reviews */}}
-					<a class="timeline-avatar{{if or .Content .Attachments (and .Review .Review.CodeComments)}} timeline-avatar-offset{{end}}"{{if gt .Poster.ID 0}} href="{{.Poster.HomeLink}}"{{end}}>
+					<a class="timeline-avatar{{if or .Content .Attachments (and .Review .Review.Content)}} timeline-avatar-offset{{end}}"{{if gt .Poster.ID 0}} href="{{.Poster.HomeLink}}"{{end}}>
```

Comment:
![image](https://github.com/user-attachments/assets/65aefc41-8d82-4f4b-bc8f-37a78fc5fde1)

Reply:
I believe they are separate from my understanding, i,.e shouldn't need to match

Avatar offset should only depends on the review `Content` (speech bubble), the below section is just logic for rendering the review comments (separate from the review content). Speech bubble only appears if content is in the review, does not depend on code comments.

i.e. Review Content:

![image](https://github.com/user-attachments/assets/69886f04-ad19-476c-8a90-28b214e1f817)

Review CodeComments:

![image](https://github.com/user-attachments/assets/9fd7f642-dd5a-4449-a0b4-e5c4bd4e978f)